lib/internal/Magento/Framework/ObjectManager/Test/Unit/Code/Generator/_files/SampleMixedProxy.txt

```diff
@@ -0,0 +1,104 @@
+namespace Magento\Framework\ObjectManager\Code\Generator;
+
+/**
+ * Proxy class for @see \Magento\Framework\ObjectManager\Code\Generator\SampleMixed
+ */
+class SampleMixed_Proxy extends SampleMixed implements \Magento\Framework\ObjectManager\NoninterceptableInterface
```

Comment:
I guess it should be
```suggest
class Proxy extends SampleMixed implements \Magento\Framework\ObjectManager\NoninterceptableInterface
```
So I expect to have a test failure now

Reply:
Test succeeded

Reply:
Right, but it shouldn't, right?

Reply:
The test for the mixed type uses a new `Sample` class called `SampleMixed` (instead of `Sample`). Compare:
https://github.com/magento/magento2/blob/de131cb51103786b55fdaeb06046a93a8bd3b480/lib/internal/Magento/Framework/ObjectManager/Test/Unit/Code/Generator/_files/SampleMixedProxy.txt

With:
https://github.com/magento/magento2/blob/de131cb51103786b55fdaeb06046a93a8bd3b480/lib/internal/Magento/Framework/ObjectManager/Test/Unit/Code/Generator/_files/SampleProxy.txt

The later is from the original (existing) test, e.g.:
- `Sample` -> `Sample_Proxy` (existing test, not using the `mixed` type at all)
- `SampleMixed` -> `SampleMixed_Proxy` (new test using the `mixed` type)